test/check-application

```diff
@@ -103,6 +112,22 @@ def run_e2e(verbose, image, browser, cpus, memory, sit):
         # Now actually run the tests
         selenium_address = "10.111.112.10" if selenium else ""
 
+        if "DEBUG_TEST" in os.environ:
+            try:
+                vnc_cmd = "vncviewer 127.0.0.2:{} &"
+                if browser == "chrome":
+                    os.system(vnc_cmd.format(fwd_port["5901"]))
+                if browser == "firefox":
+                    os.system(vnc_cmd.format(fwd_port["5902"]))
```

Comment:
Can you please use

    subprocess.call(["vncviewer", networking_selenium["forward"]["5901"]])

here? This avoids an intermediate shell, thus is safer. You can also use `check_call` if you want to get an exception on non-zero exit codes.